deployments.go

```diff
@@ -190,7 +190,16 @@ func (c *Client) DeploymentStream(ctx context.Context, deployURL string) (*webso
 	if err != nil {
 		return nil, errgo.Notef(err, "fail to generate token")
 	}
-	authString, err := json.Marshal(&AuthStruct{
+
+	header := make(http.Header)
```

Comment:
convention: We rarely use `make` in our codebase. 
```suggestion
	header := http.Header{}
```